src/AssimpWorker/model/Folder.cpp

```diff
@@ -71,19 +71,29 @@ namespace Model {
 		return childFolders;
 	}
 
+	void Folder::removeAttribute(std::string key){
```

Comment:
Noted for later refactoring (a typesafe map might be in order), for now this is not beautiful, but good enough.